### PR TITLE
Naming change for consistency GOL-108

### PR DESCRIPTION
**Pull request by salva-abate**
Branch solely dedicated to chaning names of graph objects across the project for consistency. Node is now vertex, connection is now edge, IO is now port. Branch passes all test that had been made and properly builds and runs.

### CODE REVIEW

File: lib/model/graph.js [hunk 1]
```diff
@@ -2,44 +2,44 @@
...
 
 /**
  * Creates a new graph.
  * @constructor
  *
- * @param nodes - An array containing node objects.
- * @param connections - An array containing connection objects for just
- *                      these tops level nodes.
+ * @param vertices - An array containing vertex objects.
+ * @param Edges - An array containing edge objects for just
```
Comment (RobertWarrenGilmore) [inline lib/model/graph.js]: capitalisation

File: lib/model/graphParser.js [hunk 2]
```diff
@@ -77,61 +77,61 @@ GraphParser.prototype = {
...
+    vertex.setPosition(data.x, data.y);
 
-    setIfNotExist(node.getId(), node, this._nodes);
-    return node;
+    setIfNotExist(vertex.getId(), vertex, this._vertices);
+    return vertex;
   },
 
-  // parse nodes before edges!
+  // parse vertexs before edges!
```
Comment (RobertWarrenGilmore) [inline lib/model/graphParser.js]: grammar

File: lib/model/port.js [hunk 3]
```diff
@@ -36,11 +36,9 @@ Port.prototype.getPosition = function() {
 };
 
 /**
- * @method setPos - Setter method for setting an xy-coordinate pair representing the Port's
+ * @method setPosition - Setter method for setting a position representing the Port's
  *         position in 2D space.
- * @param direction - cardinal direction (N,S,E,W) in which the port is located on the node.
- * @param percentage - percentage (from left to right or bottom to top) in which the port is located
```
Comment (jboolean) [inline lib/model/port.js]: Why were these comments removed?

Comment (jboolean) [inline lib/model/port.js]: Actually that's okay.

**Comment by RobertWarrenGilmore** [comment]
Once you address my two language notes, I approve.

**Comment by jboolean** [comment]
**Approve.**

**Comment by RobertWarrenGilmore** [comment]
I approve, too. Go, Salvador, go! Merge this puppy!

**Comment by jboolean** [comment]
You missed the style files. Not sure how you missed this, it looks totally messed up.

**Pull request merged**